magma/lib/magma/query/predicate/boolean.rb

```diff
@@ -12,6 +12,13 @@ class BooleanPredicate < Magma::ColumnPredicate
     end
 
     verb '::false' do
+      child TrueClass
+      constraint do
+        basic_constraint(@column_name, false)
```

Comment:
This seems like it might break some current things ... are there plots or manifests on Timur that rely on the behavior where `nil` records would return from a `::false` query?